Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -691,14 +691,35 @@ def get_token_request(self):
                 resp_type='response'
             )
         except Exception as e:
-            demisto.debug(f'Encountered an error for url {self._base_url}/token: {e}')
+            exception_str = str(e)
```

Comment:
Not needed

Reply:
Why not
need it to verify if the `'Incorrect user id or password'` are in the exception

Reply:
You don't need to cast it to str